documentation/docs/test-suites/semantic-interoperability/cli/configuration.md

```diff
@@ -183,3 +186,73 @@ packages/
 ```
 
 You have now successfully configured the Tier 2 test suite to test your credentials against the core UNTP data model.
+
+## Setting Up Configuration for the New Data Model
+
+The Tier 2 test suite currently supports the existing UNTP data model. To test credentials with a new data model that includes the following schemas: Conformity Credential, Digital Facility Record, Traceability Events, and Product Passport, you can generate a configuration file.
+
+Running the command `yarn run untp config` will create a configuration file containing the default values, structured as follows:
+
+```json
+{
+  "credentials": [
```

Comment:
We should update the credential types to be prefixed with "digital". But let's not stop this PR.